adapters/core2p2p/receipt.go

```diff
@@ -128,12 +127,11 @@ func AdaptExecutionResources(er *core.ExecutionResources) *gen.Receipt_Execution
 			MulMod:       uint32(er.BuiltinInstanceCounter.MulMod),
 			RangeCheck96: uint32(er.BuiltinInstanceCounter.RangeCheck96),
 		},
-		Steps:          uint32(er.Steps),
-		MemoryHoles:    uint32(er.MemoryHoles),
-		L1Gas:          AdaptFelt(l1Gas),
-		L1DataGas:      AdaptFelt(l1DataGas),
-		TotalL1Gas:     AdaptFelt(totalL1Gas),
-		TotalL1DataGas: AdaptFelt(totalL1DataGas),
```

Comment:
Remove because `total_l1_data_gas = l1_data_gas` and `total_l1_data_gas` doesn't exist in specs.